openupgrade_scripts/scripts/hr_recruitment/16.0.1.1/noupdate_changes.xml

```diff
@@ -7,121 +7,27 @@
-                    The HR Team
-                    <t t-if="'website_url' in object.job_id and hasattr(object.job_id, 'website_url') and object.job_id.website_url">
-                        Discover <a href="/jobs" style="text-decoration:none;color:#717188;">all our jobs</a>.<br/>
-                    </t>
-                </div>
-
-                <hr width="97%" style="background-color: rgb(204,204,204); border: medium none; clear: both; display: block; font-size: 0px; min-height: 1px; line-height: 0; margin: 16px 0px 16px 0px;"/>
-                <h3 style="color:#9A6C8E;"><strong>What is the next step?</strong></h3>
-                We usually <strong>answer applications within a few days</strong>.
-                <br/><br/>
-                The next step is either a call or a meeting in our offices.
-                <br/>
-                Feel free to <strong>contact us if you want a faster
-                feedback</strong> or if you don't get news from us
-                quickly enough (just reply to this email).
-                <br/>
-
-                <hr width="97%" style="background-color: rgb(204,204,204); border: medium none; clear: both; display: block; font-size: 0px; min-height: 1px; line-height: 0; margin: 17px 0px 16px 0px;"/>
-                <t t-set="location" t-value="''"/>
-                <t t-if="object.job_id.address_id.name"><strong t-out="object.job_id.address_id.name or ''">Teksa SpA</strong><br/></t>
-                <t t-if="object.job_id.address_id.street"><t t-out="object.job_id.address_id.street or ''">Puerto Madero 9710</t><br/><t t-set="location" t-value="object.job_id.address_id.street"/></t>
-                <t t-if="object.job_id.address_id.street2"><t t-out="object.job_id.address_id.street2 or ''">Of A15, Santiago (RM)</t><br/><t t-set="location" t-value="'%s, %s' % (location, object.job_id.address_id.street2)"/></t>
-                <t t-if="object.job_id.address_id.city"><t t-out="object.job_id.address_id.city or ''">Pudahuel</t>,
-                    <t t-set="location" t-value="'%s, %s' % (location, object.job_id.address_id.city)"/></t>
-                <t t-if="object.job_id.address_id.state_id.name"><t t-out="object.job_id.address_id.state_id.name or ''">C1</t>,
-                    <t t-set="location" t-value="'%s, %s' % (location, object.job_id.address_id.state_id.name)"/></t>
-                <t t-if="object.job_id.address_id.zip"><t t-out="object.job_id.address_id.zip or ''">98450</t><t t-set="location" t-value="'%s, %s' % (location, object.job_id.address_id.zip)"/></t>
-                <br/>
-                <t t-if="object.job_id.address_id.country_id.name"><t t-out="object.job_id.address_id.country_id.name or ''">Argentina</t><br/><t t-set="location" t-value="'%s, %s' % (location, object.job_id.address_id.country_id.name)"/></t>
-                <br/>
-            </div>
-          </td>
-        </tr>
-      </table>
-    </field>
-    <field name="name">Recruitment: Interest</field>
```

Comment:
Idem, comment instead of deleting. Keep the explanation you put at the end. :)

Reply:
doe